specs/iota-rs-ENGINEERING-SPEC-0000.md

```diff
@@ -85,18 +101,11 @@ for node in node_pool_urls{
 }
 ```
 
-When a return value from an API (e.g., the balance in an address) needs to be determined by quorum (i.e., not by a single healthy node only), then all the nodes in the `synced_node_list` will be queried to determine the final value.
-
 * Unsolved Questions 
```

Comment:
I think we can remove the `Unsolved Questions` here, because we already decided to use a background worker to periodically update the `synced_node_list`.

Reply:
Yes, absolutely right :)